nginx/modsecurity/apr_bucket_nginx.c

```diff
@@ -91,72 +91,10 @@ static void nginx_bucket_destroy(void *data)
     }
 }
 
-ngx_buf_t * apr_bucket_to_ngx_buf(apr_bucket *e, ngx_pool_t *pool) {
```

Comment:
because I'm not bothering to copy back from the apache structures to the nginx structures, as our use of modsec does not do anything that rewrites the request anyway